src/index.test.js

```diff
@@ -60,24 +60,15 @@ test('options: logLevel sets the log level', () => {
   })
 })
 
-test('passes on additional arguments', () => {
```

Comment:
We don't worry about this anymore. This is no longer a feature. If you want to pass arguments, put it in quotes. It's less confusing this way